src/schema/contentPreference.ts

```diff
@@ -120,6 +120,28 @@ export const typeDefs = /* GraphQL */ `
       """
       first: Int
     ): ContentPreferenceConnection!
+
+    """
+    What user blocked
+    """
+    userBlocked(
+      """
+      Id of user
+      """
+      userId: ID!
+      """
+      Entity to list (user, source..)
+      """
+      entity: ContentPreferenceType!
+      """
+      Paginate after opaque cursor
+      """
+      after: String
+      """
+      Paginate first
+      """
+      first: Int
+    ): ContentPreferenceConnection
```

Comment:
@auth?

Reply:
Yeah good point, I copied userFollow which is public.
Adding auth